packages/compose/README.md

```diff
@@ -24,6 +72,5 @@ function WrappedComponent( props ) {
 const ComponentWithInstanceIdProp = withInstanceId( WrappedComponent );
 ```
 
-Refer to each Higher Order Component's README file for more details.
+Refer to each Higher Order Component's README file for more details, see [list of components](https://github.com/WordPress/gutenberg/tree/master/packages/compose/src).
 
-<br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
```

Comment:
This footer should stay :)

Reply:
I added it back. To me it seems a bit off to have the footer, it is on the bottom of some documentation pages and not others. I don't feel all that strongly either way so if people like it, its fine.